### PR TITLE
Report Improvement

### PR DESCRIPTION
**Pull request by ShashaQin**
Hi,
Adding some new columns on the following report:
1. adding Item Group and Customer Name on Item Wise Sales History report
2. adding Item Group and Supplier Name on Item Wise Purchase History report
3. adding Item Group, Supplier, Supplier Name, Supplier Part No, Manufacturer, Manufacturer Part No, Balance Qty and Parent Website Route on Item Prices report.

### CODE REVIEW

File: erpnext/stock/report/item_prices/item_prices.py [hunk 1]
```diff
@@ -20,24 +21,47 @@ def execute(filters=None):
...
 	for item in sorted(item_map):
-		data.append([item, item_map[item]["item_name"],
-			item_map[item]["description"], item_map[item]["stock_uom"],
+		data.append([item, item_map[item]["item_name"],item_map[item]["item_group"],
+			item_map[item]["description"],
 			flt(last_purchase_rate.get(item, 0), precision),
-			flt(val_rate_map.get(item, 0), precision),
+			sn.get(item, {}).get("supplier"),
+			sn.get(item, {}).get("supplier_name"),
+			sn.get(item, {}).get("supplier_part_no"),
```
Comment (nabinhait) [inline erpnext/stock/report/item_prices/item_prices.py]: Supplier Part no is maintain in a table and hold multiple record based on supplier. What is the point to show a random supplier/supplier name / part no?

File: erpnext/stock/report/item_prices/item_prices.py [hunk 2]
```diff
@@ -20,24 +21,47 @@ def execute(filters=None):
...
+		_("Supplier Part No") + "::125",
+		_("Sales Price List") + "::80",
+		_("Purchase Price List") + "::80",
+		_("Manufacturer") + "::100",
+		_("Manufacturer Part No") + "::100",
+		_("UOM") + ":Link/UOM:80",
+		_("Balance Qty") + ":Float:80",
+		_("Valuation Rate") + ":Currency:90",
+		_("BOM Rate") + ":Currency:90",
+		_("Parent Website Route") + "::200"]
```
Comment (nabinhait) [inline erpnext/stock/report/item_prices/item_prices.py]: This is irrelevant in the context of the report

**Comment by ShashaQin** [comment]
Thanks, we'll reconsider changes on Item Prices report. Can you just merge item-wise-sales-history and item-wise-purchase-history please, or should I pull a new request?

**Comment by nabinhait** [comment]
Please send a new pull request. Closing this.

**Pull request closed without merging**